[2024-07-10T09:12:57.247+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-10T09:12:57.318+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: market_etl.create_fact_table manual__2024-07-10T09:12:14.843378+00:00 [queued]>
[2024-07-10T09:12:57.337+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: market_etl.create_fact_table manual__2024-07-10T09:12:14.843378+00:00 [queued]>
[2024-07-10T09:12:57.338+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-07-10T09:12:57.380+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): create_fact_table> on 2024-07-10 09:12:14.843378+00:00
[2024-07-10T09:12:57.403+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=143) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-10T09:12:57.411+0000] {standard_task_runner.py:63} INFO - Started process 158 to run task
[2024-07-10T09:12:57.414+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'market_etl', 'create_fact_table', 'manual__2024-07-10T09:12:14.843378+00:00', '--job-id', '296', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpsya5l405']
[2024-07-10T09:12:57.420+0000] {standard_task_runner.py:91} INFO - Job 296: Subtask create_fact_table
[2024-07-10T09:12:57.505+0000] {task_command.py:426} INFO - Running <TaskInstance: market_etl.create_fact_table manual__2024-07-10T09:12:14.843378+00:00 [running]> on host 514b016b59c3
[2024-07-10T09:12:58.070+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='market_etl' AIRFLOW_CTX_TASK_ID='create_fact_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-10T09:12:14.843378+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-10T09:12:14.843378+00:00'
[2024-07-10T09:12:58.073+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-10T09:12:58.082+0000] {python.py:237} INFO - Done. Returned value was:     SECURITY_ID BUSINESS_DATE  ...  AVERAGE_TRADED_PRICE          STOCK_ID
0          ADBL    2024-07-09  ...                285.25   ADBL_2024-07-09
1           AHL    2024-07-09  ...                518.39    AHL_2024-07-09
2          AHPC    2024-07-09  ...                154.33   AHPC_2024-07-09
3         AKJCL    2024-07-09  ...                221.13  AKJCL_2024-07-09
4          AKPL    2024-07-09  ...                163.25   AKPL_2024-07-09
..          ...           ...  ...                   ...               ...
310        USHL    2024-07-09  ...                580.62   USHL_2024-07-09
311        USLB    2024-07-09  ...               1748.07   USLB_2024-07-09
312        VLBS    2024-07-09  ...               1044.35   VLBS_2024-07-09
313       VLUCL    2024-07-09  ...                511.52  VLUCL_2024-07-09
314        WNLB    2024-07-09  ...               1511.85   WNLB_2024-07-09

[315 rows x 8 columns]
[2024-07-10T09:12:58.139+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-10T09:12:58.217+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=market_etl, task_id=create_fact_table, run_id=manual__2024-07-10T09:12:14.843378+00:00, execution_date=20240710T091214, start_date=20240710T091257, end_date=20240710T091258
[2024-07-10T09:12:58.259+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-10T09:12:58.351+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-10T09:12:58.353+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
